lib/fog/compute/kubevirt/requests/list_offlinevms.rb

```diff
@@ -5,7 +5,11 @@ module Compute
     class Kubevirt
       class Real
         def list_offlinevms(_filters = {})
-          kubevirt_client.get_offline_virtual_machines(namespace: 'default').map { |kubevirt_obj| Offlinevm.parse object_to_hash(kubevirt_obj) }
+          ovms = kubevirt_client.get_offline_virtual_machines(namespace: @namespace)
+          entities = ovms.map do
+            |kubevirt_obj| Offlinevm.parse object_to_hash(kubevirt_obj)
```

Comment:
formatting: |kubevirt_obj| should be moved to the line above

Reply:
Done